### DISCUSSION TITLE
Restore files

### DISCUSSION THREAD

**Discussion by Agentew04** [VerifyTests/Verify#1187] [Q&A] [answered]
How can I reset the state of the tests? Everytime I run a test it shows this message:
```
VerifyException : Directory: E:\Universidade\Trabalhos\SourceGenerators\Tests
Delete:
  - ObservableGeneratorSnapshotTests.GeneratePropertyCorrectly#.testProperty.g.verified.cs
  - ObservableGeneratorSnapshotTests.GeneratePropertyCorrectly.verified.txt
Equal:
  - Received: ObservableGeneratorSnapshotTests.GeneratePropertyCorrectly#ObservableAttribute.g.received.cs
    Verified: ObservableGeneratorSnapshotTests.GeneratePropertyCorrectly#ObservableAttribute.g.verified.cs
  - Received: ObservableGeneratorSnapshotTests.GeneratePropertyCorrectly#TestClass.testProperty.g.received.cs
    Verified: ObservableGeneratorSnapshotTests.GeneratePropertyCorrectly#TestClass.testProperty.g.verified.cs
```
I think the problem is that before the name of the generated file was wrong, but I don't know how to delete it or where it is.

**Comment by SimonCropp** [comment, marked as answer]
if you want to manually delete them: both those file exist in E:\Universidade\Trabalhos\SourceGenerators\Tests and u can delete them using explorer

or you can pick a tool to help u do that https://github.com/VerifyTests/Verify?tab=readme-ov-file#snapshot-management

**Reply by Agentew04** [reply]
I didn't even noticed they were there haha thanks for the help!
